<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAppDialog</class>
 <widget class="QDialog" name="AddAppDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Application</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QPushButton" name="icon_picker_button">
       <property name="toolTip">
        <string>Pick staging directory</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Application name</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>All installed mods are stored here. For hard link deployment, this has to be on the same partition as the application</string>
       </property>
       <property name="text">
        <string>Staging directory:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Icon path</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="toolTip">
        <string>Command used to run the application</string>
       </property>
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="ValidatingLineEdit" name="path_field"/>
     </item>
     <item row="0" column="1">
      <widget class="ValidatingLineEdit" name="name_field"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="icon_field"/>
     </item>
     <item row="3" column="2" rowspan="2">
      <widget class="QPushButton" name="file_picker_button">
       <property name="toolTip">
        <string>Pick staging directory</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="folder-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="command_field"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string>This will be used for FOMOD conditions</string>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="version_field"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="move_dir_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Move old staging directory to new location</string>
     </property>
     <property name="text">
      <string>Move staging directory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="import_checkbox">
     <property name="toolTip">
      <string>Create Deployers targeting the apps installation and prefix directories</string>
     </property>
     <property name="text">
      <string>Import deployers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="import_button">
       <property name="toolTip">
        <string>Import an application from steam</string>
       </property>
       <property name="text">
        <string>Import from Steam</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/validatinglineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddAppDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddAppDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
